mgrpxy/cmd/support/ptf/podman/utils_test.go

```diff
@@ -0,0 +1,262 @@
+//go:build ptf
+
+// SPDX-FileCopyrightText: 2025 SUSE LLC
+//
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
```suggestion
// SPDX-FileCopyrightText: 2025 SUSE LLC
//
// SPDX-License-Identifier: Apache-2.0
//go:build ptf
```

We have this in reverse order in other ptf files